denguefever_tw/dengue_linebot/denguebot_fsm/denguebotfsm.py

```diff
@@ -123,15 +123,19 @@ def is_selecting_zapper_func(self, event):
 
     @log_fsm_condition
     def is_selecting_bind_zapper(self, event):
```

Comment:
I don't see why these selecting functions are needed.
It seems to me they don't do anything other than their invoked functions.
Or, you do this for flexibility.
Could you please clarify this for me? Thanks!

Reply:
Oh... I misunderstood the `condconf`.
I will modify it.